tests/test_daemon.cpp

```diff
@@ -283,6 +278,11 @@ TEST_F(Daemon, data_path_with_storage_valid)
 
     EXPECT_CALL(mock_platform, multipass_storage_location()).WillOnce(Return(mp::utils::get_multipass_storage()));
 
+    ON_CALL(mock_utils, make_dir(_, _, _))
+        .WillByDefault([this](const QDir& a_dir, const QString& name, QFileDevice::Permissions permissions) {
+            return mock_utils.Utils::make_dir(a_dir, name, permissions);
+        });
+
```

Comment:
This one doesn't actually need to make a directory, so I think it would be preferable to just mock the required return, as usual. But I won't block on that.